src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -46,17 +46,14 @@
                         OnDismiss="() => ClearSelectedResource()"
                         ViewKey="ResourcesList">
         <Summary>
-            <FluentDataGrid Items="@FilteredResources" ResizableColumns="true" GridTemplateColumns="1fr 2fr 1fr 2fr 2.5fr 2fr 1fr 1fr" RowClass="GetRowClass">
+            <FluentDataGrid Items="@FilteredResources" ResizableColumns="true" GridTemplateColumns="1fr 2fr 1.02fr 2fr 2.5fr 2fr 1fr 1fr" RowClass="GetRowClass">
```

Comment:
1.02? We're not being that exact.

Is 1 too small? Increase it to 1.1.

Reply:
1.1 looks too big.